eggplant/profiles/views.py

```diff
@@ -91,6 +92,7 @@ class Profile(LoginRequiredMixin, FormView):
 
     def get_object(self, queryset=None):
         self.objects = UserProfile.objects.get(user_id=self.request.user.id)
```

Comment:
I had a look at `django.views.generic.edit` and I don't think this matters so it looks like a typo to me.

Simple typo and we keep wondering... -  there you go, CBV for you.